packages/pyright-internal/src/tests/semanticTokensProvider.test.ts

```diff
@@ -290,6 +290,21 @@ if (process.platform !== 'win32' || !process.env['CI']) {
             { type: 'function', modifiers: [], start: 87, length: 1 }, // f
         ]);
     });
+
+    test('builtins', () => {
```

Comment:
there's already a test called "builtins" (not sure why jest doesn't complain when there are duplicate test names)

i would suggest moving both tests into a `describe` block like this

```ts
describe('builtins', () => {
    test('real builtins', () => {
        // ...
    })
    test('project builtins', () => {
        // your new test here
    })
})
```

Reply:
fixed